node-graph/gcore/src/application_io.rs

```diff
@@ -91,7 +91,7 @@ impl PartialEq for ImageTexture {
 		}
 		#[cfg(not(feature = "wgpu"))]
 		{
-			true // Unit values are always equal
```

Comment:
Why did you make this change? Seems nonsensical

Reply:
Clippy

Reply:
Then just remove the compile time distinctinons, we now the same code twice which is confusing